triggers/service/BUILD.bazel

```diff
@@ -10,8 +10,9 @@ load(
 load("@build_environment//:configuration.bzl", "sdk_version")
 
 da_scala_library(
-    name = "trigger-service-lib",
+    name = "trigger-service",
     srcs = glob(["src/main/scala/**/*.scala"]),
+    tags = ["maven_coordinates=com.daml:trigger-service:__VERSION__"],
```

Comment:
This doesn’t make sense until you publish to Maven which I don’t see happening anytime soon.

Reply:
Should it come out? Does it do harm? I included to be preparatory.

Reply:
I don’t care all that much care but I could very well see this not be published to Maven at all and only included as an SDK command so it might be a bit misleading.